client/js/lounge.js

```diff
@@ -79,18 +79,45 @@ $(function() {
 		if (target.hasClass("user")) {
 			output = templates.contextmenu_item({
 				class: "user",
+				action: "whois",
 				text: target.text(),
 				data: target.data("name"),
 			});
+			output += templates.contextmenu_divider();
+			output += templates.contextmenu_item({
+				class: "action-whois",
+				action: "whois",
+				text: "Whois",
```

Comment:
"Information about this user"

Reply:
I rather keep whois here, as that is the commonly known irc terminology.

Reply:
And I'd... rather not 😅
New users would have no idea what this is, while experienced users would feel free to type `/whois` anyway.
You could also do "Information about this user (`whois`)" if you reaaally want to keep "whois" in there. I'd be okay with that.

Reply:
Would you be okay with just "User information"?  Fits much better with the width of the menu.

Reply:
How is "whois" confusing? It's literally saying it in the name what it is.

I'm really not a fan of these long sentences you want in the context menu.

How about "Who is this user?" or something? Then you get the whois in, but it's a bit more of a sentence for new users.

Reply:
(I'd definitely be fine with "User Information", that works fine for me)

Reply:
User information is great, thanks! And yeah, "whois" doesn't say much unless you already know what a whois is in the context of IRC. "User information" (I was also fine with "User information (whois)") is way clearer :D

Re: sentences, I totally understand. It's difficult to give a proper explanation in 3 words max, but yeah my suggestion was too long. Thanks all!